test/adapters/prebid.spec.js

```diff
@@ -991,6 +999,26 @@ describe('ADAPTER: Prebid', function() {
             delete CONF.adapters[adapterID][CONSTANTS.CONFIG.SERVER_SIDE_ENABLED];
             done();
         });
+
+        xit('should set config for single request partners present in constants',function(done){
+            var expectedResult = {
+                "rubicon":{
+                    singleRequest:true
+                }
+            }
+            PREBID.fetchBids(activeSlots, impressionID);
+            window["owpbjs"].setConfig.should.be.called.with(expectedResult);
+            var prebidConfig = window["owpbjs"].getConfig();
+            expectedResult.should.be.equal(prebidConfig);
```

Comment:
we can check if the required config is present in the generated prebid config

Reply:
How to access prebidConfig generated in fetchBids ?

Reply:
I passed config in setConfig and stub method to set config in a variable
Introduced a method getConfig which returns this config and checked against each other